defaults/main.yml

```diff
@@ -4,6 +4,7 @@ rhc_auth: {}
 rhc_baseurl: null
 rhc_insights:
   state: present
+  remediation: present
```

Comment:
since there are other changes to do... a small nitpick here: can you please place it alphabetically sorted within `rhc_insights`? :)